README.md

```diff
@@ -490,6 +497,22 @@ instance.get('/longRequest', {
 });
 ```
 
+```js
+// Create an instance using the config defaults provided by the library
+// At this point the params config value is undefined as is the default for the library
+const instance = axios.create();
+
+// Override the params default for the library
+// Now all requests using this instance will pass the param defined below
+instance.defaults.params = { q: 'some-value' }
+
+// The `params` object will be entirely overridden, and the
+// previously defined key `q` will no longer exist
+instance.get('/testRequest', {
+  params: { a: 'a-new-value' }
+})
+```
```

Comment:
Thanks for your sweet examples. But I am afraid that `params` will be changed with #2190.